### ISSUE TITLE
I cannot compile proton bride

### ISSUE THREAD

**Issue by doggodanubus** [ProtonMail/proton-bridge#258]
Issue tracker is ONLY used for reporting bugs with technical details. "It doesn't work" or new features should be discussed with our customer support. Please use bug report function in Bridge or contact bridge@protonmail.ch.

I get a GOROOT is set to GOPATH error. 

## Expected Behavior

It compiles

## Current Behavior

Crash screaming about GOPATH is same as GOROOT

## Possible Solution

I tried setting GOPATH to $HOME as that is where GO is 
## Steps to Reproduce

1. download the git repository
2. cd to the proton-bridge directory and type make and it will crash as go is not available
3. download go1.13.15 and unpack it. I put it in the $HOME folder as I recall that is where it had once been
4. cd into proton-bridge and make and it will give the GOPATH error, if you set it to $HOME it crashes later.

## Version Information

Current git as of today 

## Context (Environment)

Get a working bridge from code I compiled. 
## Detailed Description

No idea

**Comment by doggodanubus** [comment]
Set your $PATH to include the path to the go executable folder.

**Comment by doggodanubus** [comment]
Ok I've read all instructions I can and found that go1.13.4 is expected and gets past some errors. 
I'm attaching a file of the build output. Relevant lines are below where it squeals like a piggy and dies because your code is old.

`
[buildresults_2.10_London_with_go1.13.4.txt](https://github.com/ProtonMail/proton-bridge/files/8009347/buildresults_2.10_London_with_go1.13.4.txt)
# github.com/ProtonMail/proton-bridge/pkg/pmapi
pkg/pmapi/messages.go:330:24: signature.Hash.String undefined (type "crypto".Hash has no field or method String)
note: module requires Go 1.15

# github.com/therecipe/qt/widgets
widgets-minimal.cpp: In function ‘void* QDesktopWidget_Screen(void*, int)’:
widgets-minimal.cpp:774:57: warning: ‘QWidget* QDesktopWidget::screen(int)’ is deprecated: Use QScreen [-Wdeprecated-declarations]
  774 |         return static_cast<QDesktopWidget*>(ptr)->screen(screen);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
In file included from /tera/sourcecode/proton-bridge/vendor/github.com/therecipe/env_linux_amd64_513/5.13.0/gcc_64/include/QtWidgets/QDesktopWidget:1,
                 from widgets-minimal.cpp:16:
/tera/sourcecode/proton-bridge/vendor/github.com/therecipe/env_linux_amd64_513/5.13.0/gcc_64/include/QtWidgets/qdesktopwidget.h:77:45: note: declared here
   77 |     QT_DEPRECATED_X("Use QScreen") QWidget *screen(int screen = -1);
      |                                             ^~~~~~

make: *** [Makefile:122: cmd/Desktop-Bridge/deploy/linux/proton-bridge] Error 1'

**Comment by doggodanubus** [comment]
I may have missed some instructions. I have it working. Here's the results.  Ok seem to have done something with it. The hash function needs GO 1.15.  

Use therecipe wiki command to put go in /usr/local but specify 1.15 for go. It will set your .profile but I use .bashrc so I fixed that

I modified vendor/github.com/therecipe/qt/widgets/minimal_cgo_linux_linux_amd64.go with -Wno-deprecated everywhere it looked right. This was useless as stated below.  It compiled and squalled about libsecret-1, used my distros libsecret and it worked. 

But now that file is back to unmodified so it probably wasn't wasn't the deprecation causing an issue. I was not seeing the libsecret error.  I went back and looked at the file I uploaded and *headdesk* right there is the libsecret error a page up. 

A quick test shows it is all Puppies and encrypted Rainbow bridges.
